### ISSUE TITLE
Automated accessibility tests

### ISSUE THREAD

**Issue by colinmurphy01** [usdoj-crt/beta-ada#29]
### Story
As a developer or content manager deploying a new feature or content to a branch, I'd like to know if and where the content or feature doesn't meet WCAG 2.1(?) and AA standards when running a build.

### Considerations
- When should this test run and should a preview be generated for manual review?
- Should this test run for every branch / update?
- How will the developer or content manager receive the evaluation and be able to improvements?

### When this issue is done
- [ ] TBD
